qtop.py

```diff
@@ -11,9 +11,9 @@
 from optparse import OptionParser
 import datetime
 from collections import Counter, OrderedDict
-import os
-import re
-import yaml
+# import os
+# import re
+# import yaml
```

Comment:
not yet convinced i don't need them (they *should* be in use!). Also, don't know how to revert as quick as I can uncomment